rubix_admin/daemon.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Same here. Is this an editor setting in your IDE ?

Reply:
Maybe. I will fix that. Thanks.